handler/routes.go

```diff
@@ -978,10 +978,13 @@ func Status(db store.IStore) echo.HandlerFunc {
 						LastHandshakeTime: devices[i].Peers[j].LastHandshakeTime,
 						LastHandshakeRel:  time.Since(devices[i].Peers[j].LastHandshakeTime),
 						AllocatedIP:       allocatedIPs,
-						Endpoint:          devices[i].Peers[j].Endpoint.String(),
 					}
 					pVm.Connected = pVm.LastHandshakeRel.Minutes() < 3.
 
+					if isAdmin(c) {
+						pVm.Endpoint = devices[i].Peers[j].Endpoint.String()
+					}
+
```

Comment:
I actually don't want to dive into the user privilege topic. As this is a read-only operation, I'd like to keep it simple, and available to any user who can access the status page.